src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from "@angular/core";
+import { GlobalDataService } from "src/app/globalData.service";
+import { IGlobal } from "src/app/global";
+import { GoogleChartInterface } from "ng2-google-charts";
+
+@Component({
+  selector: "app-home",
+  templateUrl: "./home.component.html",
+  styleUrls: ["./home.component.scss"],
+})
+export class HomeComponent implements OnInit {
+  NewConfirmed = 0;
+  TotalConfirmed = 0;
+  TotalDeaths = 0;
+  TotalRecovered = 0;
+  Date = "";
+  Country = "";
+  data: IGlobal[];
+  pieChart: GoogleChartInterface = {
+    chartType: "PieChart",
+  };
+  lineChart: GoogleChartInterface = {
+    chartType: "LineChart",
+  };
+  isShowDivIf = true;
+
+  toggleDisplayDivIf() {
+    this.isShowDivIf = !this.isShowDivIf;
+  }
+  constructor(private dataService: GlobalDataService) {}
+
+  initChart() {
+    let datatable = [];
+    datatable.push(["Country", "Cases"]);
```

Comment:
No need to create an empty array and on the line bellow populate it with data. Use a one-liner for this, since you are adapting this approach